content/docs/command-reference/add.md

```diff
@@ -144,6 +144,9 @@ directory symlinks.
 - `--external` - allow `targets` that are outside of the DVC repository. See
   [Managing External Data](/doc/user-guide/managing-external-data).
 
+- `--glob` - allows adding `targets` that match the specified pattern.
+  Shell-style wildcards are supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`.
```

Comment:
How about
```suggestion
- `--glob` - enables `targets` to include path patterns. Wildcards supported:
  `*`, `?`, `[seq]`, `[!seq]`, `**`, etc. See
  [Unix style pathname pattern expansion](https://docs.python.org/3/library/glob.html)
  for details.
```

Reply:
Sorry guys, I realize you have already worked this and I just came with a different suggestion based on https://github.com/iterative/dvc/pull/4864#pullrequestreview-528893719. Please ignore this one for now, I'll apply it later if needed.